docs/Settings.md

```diff
@@ -57,4 +57,19 @@ The settings can be saved to and loaded from an XML file using the following met
   fun loadStateFromFile(project: Project, filePath: String)
   ```
 
-These methods allow you to persist configuration data by serializing `MinimizationPluginState` to XML and then deserializing it back into the application.
\ No newline at end of file
+These methods allow you to persist configuration data by serializing `MinimizationPluginState` to XML and then deserializing it back into the application.
+
+### How to add new minimization stage in settings
+
+1. In [MinimizationPluginState][plugin-state] add your stage to ```defaultStages```
```

Comment:
And `elementType` parameter of the `@get:Xcollection` annotation, I suppose

Reply:
Yes, add this one, and we can merge